pur/__init__.py

```diff
@@ -330,7 +337,7 @@ def _get_requirements_and_latest(filename, force=False, minor=[], patch=[],
     """
     session = PipSession()
     finder = PackageFinder(
-        session=session, find_links=[], index_urls=[PyPI.simple_url])
+        session=session, find_links=[], index_urls=index_url)
```

Comment:
We need to move `PackageFinder()` init into a function so it's passed `index_urls` after being parsed from command line arguments.